src/components/lab/NumberInput/index.ts

```diff
@@ -0,0 +1,5 @@
+export {NumberInput} from './NumberInput';
+export type {NumberInputProps} from './NumberInput';
+
+export {getNumericInputValidator} from './utils';
+export type {GetNumericInputValidatorParams} from './utils';
```

Comment:
Aren't these internal helpers? Why we export them?

Reply:
No, they are not used in component. This utility function is for external use only. NumberInput itself doesn't show any validation errors

Reply:
Then why keep this code? We cannot maintain it properly without specific usecases